cetmix_tower_server/models/cx_tower_variable_value.py

```diff
@@ -32,10 +32,21 @@ class TowerVariableValue(models.Model):
         inverse="_inverse_is_global",
         store=True,
     )
-
-    value_char = fields.Char(string="Value")
     note = fields.Text(related="variable_id.note", readonly=True)
     active = fields.Boolean(default=True)
+    has_options = fields.Boolean(
```

Comment:
We have 'variable_type' field in the variable now, so we can use it instead.

Reply:
I will change it